### DISCUSSION TITLE
Getting `Error while parsing entering body: Unexpected end of JSON input.` message, don't know what's wrong...

### DISCUSSION THREAD

**Discussion by Zcehtro** [mockoon/mockoon#900] [Help] [answered]
I'm getting the `Error while parsing entering body: Unexpected end of JSON input.` message when hitting an endpoint on the API. The content is generated in a databucket with faker. This is what I have in the `accounts` databucket:

```text
{
  "list": [
    {{#repeat 8}}
    {
      "code": "{{faker 'finance.account'}}",
      "description": "{{faker 'finance.accountName'}}"
    }
    {{/repeat}}
  ]
}
```

And the endpoint is configured like this:

```text
{{data 'accounts'}}
```

Does somebody know why this is happening?

**Comment by Zcehtro** [comment, marked as answer]
Found the error. I was using hard tabs to indent instead of spaces. Message is gone. 

Would be nice to get a warning in the UI when this is happening. Sometimes users might just copy&paste code and won't know what's wrong, especially if the problem is in whitespaces.

**Reply by 255kb** [reply]
The only issue is that this error is due to a JSON syntax error inside the request body you sent to mockoon, not the response you prepared in mockoon. The entering body has to be valid JSON to be parsed properly and we are reporting the error we get from the JSON parse function.

**Comment by Java-Coffee** [comment]
邮件已收到，谢谢！
